app/indexers/image_indexer.rb

```diff
@@ -0,0 +1,21 @@
+# Generated via
```

Comment:
Catch up with Hyrax's work generator, preserving earlier work to make Images use the IIIFThumbnailPathService.